spec/features/admin/admin_edits_user_spec.rb

```diff
@@ -14,7 +14,7 @@
     fill_in "user_email", with: "Mary@example.com"
     fill_in "user_email_confirmation", with: "Mary@example.com"
     fill_in "user_password", with: user.password
-    click_on I18n.t("admin.users.edit.submit")
+    click_on I18n.t("helpers.submit.user.update")
```

Comment:
This does not seem to belong in this PR :/

Reply:
Sorry, clearing out some noise and forgot to change it back - done in sha: d4e1ee7